File: src/main/java/mediathek/gui/dialog/about/AboutDialog.java
package mediathek.gui.dialog.about;

import com.jidesoft.dialog.ButtonPanel;
import com.jidesoft.dialog.StandardDialog;
import mediathek.tool.EscapeKeyHandler;

import javax.swing.*;
import java.awt.*;

public class AboutDialog extends StandardDialog {
    public AboutDialog(Frame owner) {
        super(owner,"Über dieses Programm", true);
        setDefaultCloseOperation(JDialog.DISPOSE_ON_CLOSE);
        EscapeKeyHandler.installHandler(this, this::dispose);

        //setResizable(false);
        pack();
    }

    @Override
    public JComponent createBannerPanel() {
        return null;
    }

    @Override
    public JComponent createContentPanel() {
        return new AboutDialogPanel();
    }

    @Override
    public ButtonPanel createButtonPanel() {
        return null;
    }
}

File: src/main/java/mediathek/gui/dialog/about/AboutDialog.kt
package mediathek.gui.dialog.about

import javafx.embed.swing.JFXPanel
import javafx.scene.Scene
import mediathek.javafx.tool.JavaFxUtils
import mediathek.tool.EscapeKeyHandler
import java.awt.BorderLayout
import java.awt.Frame
import javax.swing.JDialog
import javax.swing.SwingUtilities

class AboutDialog(owner: Frame?) : JDialog(owner, "Über dieses Programm", true) {
    init {
        defaultCloseOperation = DISPOSE_ON_CLOSE
        isResizable = false
        EscapeKeyHandler.installHandler(this) { dispose() }
        contentPane.layout = BorderLayout()
        val fxPanel = JFXPanel()
        contentPane.add(fxPanel, BorderLayout.CENTER)
        JavaFxUtils.invokeInFxThreadAndWait {
            fxPanel.scene = Scene(AboutController())
            SwingUtilities.invokeLater { pack() }
        }
    }
}